todo_app/app.py

```diff
@@ -1,11 +1,41 @@
-from flask import Flask
+from flask import Flask, redirect, url_for, render_template, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_item, get_items, add_item, save_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    todo = get_items()
+    print(todo)
+    todo = sorted(todo, key=lambda x: x['status'], reverse=True)
+    print(todo)
+    return render_template('index.html', todo=todo)
+
+@app.route('/todo/submit', methods=['POST'])
+def add_todo():
+    add_item(request.form.get('todo-title'))
+
+    return redirect(url_for('index'))
+
+@app.route('/todo/update', methods=['POST'])
+def update_todo():
+    todo = get_items()
+
+    for item in todo:
+        form_item = dict(request.form)
+        id = str(item['id'])
+        if id in form_item:            
+            item['status'] = 'Completed'
+        else:            
+            item['status'] = 'Not Started'
+
+        save_item(item)
+
+    return redirect(url_for('index'))
+
+if __name__ == '__main__':
+    app.run()
```

Comment:
Very minor, but you should find you don't _need_ this line if we're consistently running the application with `poetry run flask run`, and in that case I'm a fan of getting rid of it so that we encourage everyone to treat the app consistently, and avoid having to deal with the natural overheads of handling multiple subtly different approaches
```suggestion
```